camp_fin/urls.py

```diff
@@ -104,6 +105,7 @@
         name="expenditure-detail",
     ),
     path("committees/", CommitteeList.as_view(), name="committee-list"),
+    path("downloads/", downloads_redirect_view, name="downloads"),
```

Comment:
i think you can just do

```python
path("downloads/",
     HttpResponsePermanentRedirect(reverse("camp-fin-downloads")),
     name="downloads"),
```

seems clearer than creating a custom view that's just wrapping that up.

Reply:
Agreed. Doing it this way threw an error, so I used a [RedirectView](https://docs.djangoproject.com/en/5.1/ref/class-based-views/base/#redirectview).